tests/functional/amo_sign_up.js

```diff
@@ -18,22 +30,14 @@ define([
     },
 
     'as a migrating user': function () {
-      return FunctionalHelpers.openFxaFromRp(this, 'signup', MIGRATE_PARAMS)
-        .then(FunctionalHelpers.visibleByQSA('.info.nudge'))
-        .findByCssSelector('.info.nudge a')
-          .click()
-        .end()
-
-        .findByCssSelector('#fxa-signin-header')
-        .end()
-
-        .findByCssSelector('input[type="email"]')
-        .getAttribute('value')
-        .then(function (resultText) {
-          // check the email address is empty
-          assert.equal(resultText, '');
-        })
-        .end();
+      return this.remote
+        .then(openFxaFromRp(this, 'signup', { query: QUERY_PARAMS }))
+        .then(visibleByQSA('.info.nudge'))
+        .then(click('.info.nudge a'))
+
+        .then(testElementExists('#fxa-signin-header'))
+
```

Comment:
Putting my "King Of The Bikeshed" hat on for a moment, these blank lines seem unnecessary now that all of the test steps are one-liners.